src/codegate/config.py

```diff
@@ -209,11 +215,17 @@ def from_env(cls) -> "Config":
             if "CODEGATE_SERVER_KEY" in os.environ:
                 config.server_key = os.environ["CODEGATE_SERVER_KEY"]
             if "CODEGATE_FORCE_CERTS" in os.environ:
-                config.force_certs = os.environ["CODEGATE_FORCE_CERTS"]
+                config.force_certs = cls.__bool_from_string(os.environ["CODEGATE_FORCE_CERTS"])
```

Comment:
Fix for a bug I spotted while working on this part of the code: `force_certs` expects a bool, but in this line we assign a string to the value. The likely consequence of this is that if someone explicitly sets this value to `false` in an env var, it will end up evaluating to `True` when treated as a boolean.